doc/40_Configuration.md

```diff
@@ -50,42 +51,40 @@ image: node:alpine
 
 If you want to access services running in your workspace, e.g. a development HTTP server on port `8080`,
 you need to expose that port first. Gitpod has two means of doing that:
- 1. On-the-fly: when you start a process which listens to a port in your workspace, Gitpod will ask you
-    if you want to expose that port to the internet.
- 2. In your configuration: if you already know that you want a particular port exposed, you can configure it
-    in the `.gitpod` file and skip the extra click later on. For example:
+  1. On-the-fly: when you start a process which listens on a port in your workspace, Gitpod will ask you
+     if you want to expose that port to the internet.
+  2. In your configuration: if you already know that you want a particular port exposed, you can configure
+     it in the `.gitpod` file and skip the extra click later on. For example:
 ```yaml
 ports:
   - port: 8080
     protocol: "http"
 ```
-Ports are mapped to their own URLs prefixing the workspace URL with `{portnumber}-`. For
-instance: `https://8080-fe76ea5b-924d-4a67-a2d5-24a259619fa7.ws.gitpod.io/`. At the moment you can only expose
+Ports are mapped to their own URLs, prefixing the workspace URL with `{portnumber}-`. For instance:
+`https://8080-fe76ea5b-924d-4a67-a2d5-24a259619fa7.ws.gitpod.io/`. At the moment you can only expose
 HTTP servers.
 
 ## Start Script
 
-In many cases it makes sense to start the build and maybe something like a dev server upon workspace startup.
-To that end you can provide a shell command which is executed in a terminal on start.
+In many cases it makes sense to start the build and maybe something like a dev server upon workspace
+initialization. To that end you can provide a shell command that gets executed in a terminal on start.
 
 For instance, the start script for the Gitpod documentation repository is defined as:
 ```yaml
 tasks:
   - command: "npm install && npm run serve"
 ```
-
-Similarly to Docker, you can chain multiple shell commands with `&&`.
+Note that you can chain multiple shell commands with `&&`.
 
 ## Working with Go
 
-Go projects need a [specific workspace layout](https://golang.org/doc/code.html#Organization).
-For a Go project, source code of your repository and its dependencies needs to be in the directories
+Go projects need a [specific workspace layout](https://golang.org/doc/code.html#Organization):
+source code of your repository and its dependencies must be in the directories
```

Comment:
is this missing a `the`, as in `the source code of your repository ...`?

Reply:
I agree, will fix that